packages/components/tag/src/utils/constants.js

```diff
@@ -0,0 +1,18 @@
+export const types = {
+	SUCCESS: 'success',
+	WARNING: 'warning',
+	WARNING_LITE: 'warning-lite',
+	DANGER: 'danger',
+	BLUE: 'blue',
+	BLUE_LITE: 'blue-lite'
+};
+
+export const labels = {
+	SINGLE_LABEL: ['Label 1'],
+	MULTIPLE_LABELS: ['Label 1', 'Label 2']
+};
+
+export const values = {
+	SINGLE_VALUE: ['value 1'],
+	MULTIPLE_VALUES: ['value 1', 'value 2']
+};
```

Comment:
I think we should not use the utils for the stories-only usage and everything related to the stories should live inside that directory. In this case I would put it directly to the happo file.

Reply:
I agree, I just thought it was a decided standard. I also think it's over the top in a case like this.